testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -191,8 +191,17 @@
   fill_in('command', with: cmd, fill_options: { clear: :backspace })
 end
 
-When(/^I enter target "([^"]*)"$/) do |minion|
-  fill_in('target', with: minion, fill_options: { clear: :backspace })
+When(/^I enter target "([^"]*)"$/) do |host|
+  value =
+    if $is_cloud_provider
+      get_system_name(host)
+    elsif host == 'ceos_minion'
+      "*centos*"
+    elsif host == 'ubuntu_minion'
+      "*ubuntu*"
+    end
+
+  fill_in('target', with: value, fill_options: { clear: :backspace })
```

Comment:
This is wrong, as it doesn't consider other cases for this step.
What happen if I want to provide there a `host = 'sle_minion'` for instance?